src/main/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/config/ApplicationInsightsConfiguration.kt

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.digital.hmpps.crimeportalgateway.config
```

Comment:
See my earlier comments on the Condition.
This file looks like it is doing what `TelemetryConfig` is supposed to do. We should either fix that and delete this, or just delete that as appropriate